src/sysext/sysext.c

```diff
@@ -51,6 +51,13 @@
 #include "varlink-io.systemd.sysext.h"
 #include "verbs.h"
 
+typedef enum MutableMode {
+        MUTABLE_YES,
+        MUTABLE_NO,
+        MUTABLE_AUTO,
+        MUTABLE_IMPORT,
```

Comment:
we almost always do at the end:

```c
_MUTABLE_MODE_MAX,
_MUTABLE_MODE_INVALID = -EINVAL,
```
i'd just do that here too

Reply:
Done.